app/src/main/java/com/futurice/freesound/feature/common/ui/adapter/RecyclerViewAdapter.java

```diff
@@ -38,10 +38,10 @@
 /**
  * Implementation of {@link RecyclerView.Adapter} for {@link DisplayableItem}.
  */
-public final class RecyclerViewAdapter extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
+public final class RecyclerViewAdapter<T> extends RecyclerView.Adapter<RecyclerView.ViewHolder> {
```

Comment:
Ah, I didn't know you could do that on kotlin. Sounds promising. I'll finish the review later